### PR TITLE
#169392403: view comments of a particular article

### PR DESCRIPTION
**Pull request by habinezadalvan**
#### What does this PR do?
- fetches all comments of an article
#### Description of Task to be completed?
- completed the functionality of fetching all comments of a particular article.
#### How should this be manually tested?
- clone the repo
- run:
 **git checkout bg-fix-view-all-comments-of-a-particular-article-169392403**
**npm install**
**npm run dev**
- use postman and do the following:
**Create an article** and user the article slug to comment
Method: POST
url: **http://localhost:3000/api/v1/comments/:articleSlug**
body: {"body": "your comment"}
#### Any background context you want to provide?
none
#### What are the relevant pivotal tracker stories?
[169392403](https://www.pivotaltracker.com/story/show/169392403)
#### Screenshots (if appropriate)
Screen Shot 2019-10-28 at 14 09 32

#### Questions:
none

### CODE REVIEW

**Review by nshutijonathan — CHANGES_REQUESTED** [review]
Seems you did not test your controller but the coverage doesn't drop too much

**Review by nshutijonathan — DISMISSED** [review]

**Review by minega25 — DISMISSED** [review]

**Review by salviosage — CHANGES_REQUESTED** [review]
please fix Travis issues

File: src/routes/api/comment/comments.route.js [hunk 1]
```diff
@@ -6,8 +6,11 @@ import confirmEmailAuth from '../../../middlewares/emailVarification.middleware'
 
 const router = express.Router();
 router.post('/:slug', [auth, CommentsValidation], Comments.createComment);
+router.get('/:articleSlug', Comments.getAllCommentsOfArticle);
 router.get('/', [auth], Comments.getComments);
 router.delete('/:id', [auth], Comments.deleteComment);
+router.delete('/particularArticle/:id', [auth], Comments.deleteCommentOfAnArticle);
```
Comment (salviosage) [inline src/routes/api/comment/comments.route.js:12]: I don't think delete have a relationship with delete, so you might still use the current delete comment endpoint

**Review by salviosage — CHANGES_REQUESTED** [review]
nice work I have left some comment

**Review by minega25 — APPROVED** [review]
LGTML

**Review by salviosage — APPROVED** [review]
LGTM

**Pull request merged**